src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,44 @@
+package duke.tasks;
+
+/**
+ * Stores inputs of tasks
+ */
+public class Task {
+    private final String name;
+    private boolean checkMark;
+
```

Comment:
Perhaps the `checkMark` boolean could be renamed to better suit the coding standards.